Sources/OktaOidc/Common/Internal/OIDAuthorizationService+Okta.swift

```diff
@@ -35,33 +35,43 @@ extension OKTAuthorizationService {
 
             delegate?.didReceive(response)
             guard let response = response as? HTTPURLResponse else {
-                callback(nil, error)
+                callback(nil, OktaOidcError.api(message: "Authentication Error: No response", underlyingError: nil))
```

Comment:
You forgot to pass the `error` in.
```suggestion
                callback(nil, OktaOidcError.api(message: "Authentication Error: No response", underlyingError: error))
```